lib/creator/yeoman/webpack-generator.js

```diff
@@ -1,22 +1,141 @@
+/* eslint-disable quotes */
 const Generator = require('yeoman-generator');
-const Input = require('webpack-addons').Input;
-
+const assetFilterFunction = require('webpack-addons').assetFilterFunction;
+const externalRegExpFunction = require('webpack-addons').externalRegExpFunction;
+const parseValue = require('webpack-addons').parseValue;
 
 module.exports = class WebpackGenerator extends Generator {
 	constructor(args, opts) {
 		super(args, opts);
-		this.configuration = {};
+		this.configuration = {
+			webpackOptions: {},
+			topScope: []
+		};
 	}
 	prompting() {
-		return this.prompt([Input('entry', 'What is the name of the entry point in your application?'),
-			Input('output', 'What is the name of the output directory in your application?')]).then( (answer) => {
-				this.configuration.webpackOptions = answer;
-			});
-	}
-	config() {}
-	childDependencies() {
-		this.configuration.childDependencies = ['webpack-addons-preact'];
+		this.configuration.webpackOptions.entry = {
```

Comment:
Removed for now, testing from addon perspective, will add a full scaffold for this later.